graphblas/core/vector.py

```diff
@@ -795,6 +795,118 @@ def from_pairs(cls, pairs, dtype=None, *, size=None, dup_op=None, name=None):
             )
         return cls.from_coo(indices, values, dtype, size=size, dup_op=dup_op, name=name)
 
+    @classmethod
+    def from_dense(cls, values, dtype=None, *, size=None, name=None):
```

Comment:
`missing_value=None` to indicate which value to drop.

Reply:
This will allow us to replace `io.from_numpy` with this implementation.

Reply:
Using `select.valuene` after importing from dense seems to work well.